packages/odyssey-react-mui/src/icons.generated/index.ts

```diff
@@ -26,6 +26,8 @@ export * from "./ArrowUnsorted";
 export * from "./ArrowUp";
 export * from "./ArrowUpperLeft";
 export * from "./ArrowUpperRight";
+export * from "./Aura";
+export * from "./AuraWordmark";
```

Comment:
plmk if we this exists already or if I should combine these 2 into a single svg/icon.